# ==============================================================================
# REGRESSOR COMMIT DIFF
# ==============================================================================
# File: testing/web-platform/meta/fedcm/fedcm-disconnect.sub.https.html.ini
# Commit: da37e2d8a175
# Full Hash: da37e2d8a175fda08a98920d820dda0899b6aedf
# Author: Benjamin VanderSloot <bvandersloot@mozilla.com>
# Date: 2025-06-13 09:36:48
# Regressor Bug: 1969017
# File Overlap Count: 2
# Description:
#   Bug 1969017 - Refactor FedCM to use a managed actor rather than tagging onto WindowGlobalParent - r=anti-tracking-reviewers,emz
#   
#   Differential Revision: https://phabricator.services.mozilla.com/D251617
# ==============================================================================

diff -r 15b92f98f62c -r da37e2d8a175 testing/web-platform/meta/fedcm/fedcm-disconnect.sub.https.html.ini
--- a/testing/web-platform/meta/fedcm/fedcm-disconnect.sub.https.html.ini	Thu Jun 12 18:16:52 2025 +0000
+++ b/testing/web-platform/meta/fedcm/fedcm-disconnect.sub.https.html.ini	Thu Jun 12 18:16:53 2025 +0000
@@ -1,35 +1,22 @@
 [fedcm-disconnect.sub.https.html]
-  expected:
-    if os == "win": TIMEOUT
+  expected: [OK, TIMEOUT]
   [Repeatedly calling disconnect should eventually fail]
     expected: [FAIL, TIMEOUT, NOTRUN]
 
   [Test that disconnect fails when there is no account to disconnect]
-    expected:
-      if os == "win": [TIMEOUT, FAIL, NOTRUN]
-      [PASS, FAIL, TIMEOUT, NOTRUN]
+    expected: [PASS, FAIL, TIMEOUT, NOTRUN]
 
   [Test that disconnect succeeds when there is an account to disconnect]
-    expected:
-      if os == "win": [NOTRUN, FAIL, TIMEOUT]
-      [FAIL, TIMEOUT, NOTRUN]
+    expected: [FAIL, TIMEOUT, NOTRUN]
 
   [Test that disconnecting the same account twice results in failure.]
-    expected:
-      if os == "win": [NOTRUN, FAIL, TIMEOUT]
-      [FAIL, TIMEOUT, NOTRUN]
+    expected: [FAIL, TIMEOUT, NOTRUN]
 
   [Disconnect passing an incorrect ID can still succeed]
-    expected:
-      if os == "win": [NOTRUN, FAIL, TIMEOUT]
-      [FAIL, TIMEOUT, NOTRUN]
+    expected: [FAIL, TIMEOUT, NOTRUN]
 
   [Disconnect is bound to each IDP]
-    expected:
-      if os == "win": [NOTRUN, FAIL, TIMEOUT]
-      [FAIL, TIMEOUT, NOTRUN]
+    expected: [FAIL, TIMEOUT, NOTRUN]
 
   [Test that disconnect succeeds when there is a pending get request and the get request succeeds after the disconnect]
-    expected:
-      if os == "win": [NOTRUN, FAIL, TIMEOUT]
-      [FAIL, TIMEOUT, NOTRUN]
+    expected: [FAIL, TIMEOUT, NOTRUN]